nibabel/dataobj_images.py

```diff
@@ -191,11 +204,155 @@ def get_data(self, caching='fill'):
             self._data_cache = data
         return data
 
+    def get_fdata(self, caching='fill', dtype=np.float64):
```

Comment:
any reason why float64 over float32 is the default?

Reply:
@satra - the default up till now has been to return float64 for images with scaling.  Also, some images do have float64 and it seems a shame to downgrade them with this call.  So float32 seems to me like an option rather than a default.